tests/saw/Makefile

```diff
@@ -30,6 +30,7 @@ export LIBCRYPTO_ROOT:=$(shell pwd)/../../libcrypto-root
 all:
 	@echo "Running formal verification with ${YICES_VERSION}"
 	@${MAKE} clean-logs
+	@${MAKE} clean_the_dir
```

Comment:
isn't `${MAKE} clean_the_dir` in the `bike_bitcode` target enough?

Reply:
It can be, but If you run then `make bike` then `make` in sequence it will not recompile the bitcodes for bike and the build will fail. I find it somewhat annoying, but we can leave it without if you prefer.